### PR TITLE
README: add summary of project status

### PR DESCRIPTION
**Pull request by cyphar**
We should forward users to umoci (which is now part of the OCI), because
of the many issues with oci-image-tools in the area of image
manipulation. Sadly there is no obvious home to move the validation
code, so we can just tell users to be aware that the validation code is
not being actively maintained.

/cc @opencontainers/image-tools-maintainers
Signed-off-by: Aleksa Sarai <cyphar@cyphar.com>

### CODE REVIEW

**Comment by cyphar** [comment]
@opencontainers/image-tools-maintainers Can we please get this merged so people aren't confused about what tools the OCI recommends using?

**Review by vbatts — APPROVED** [review]
LGTM

**Comment by cyphar** [comment]
LGTM -- let me see if I can self-LGTM...

**Review by coolljt0725 — APPROVED** [review]
LGTM

**Pull request closed without merging**
